core/src/main/scala/org/bitcoins/core/protocol/blockchain/ChainParams.scala

```diff
@@ -399,6 +438,57 @@ object RegTestNetChainParams extends BitcoinChainParams {
   override lazy val network: BitcoinNetwork = RegTest
 }
 
+object SigNetChainParams extends BitcoinChainParams {
+  override lazy val networkId = "signet"
+  override lazy val genesisBlock: Block =
+    createSignetGenesisBlock(UInt32(1534313275),
+      UInt32(621297),
+      UInt32(0x1e2adc28),
+      Int32.one,
+      Satoshis(5000000000L))
+
+  override lazy val base58Prefixes: Map[Base58Type, ByteVector] =
+    Map(
+      Base58Type.PubKeyAddress -> hex"7d",
+      Base58Type.ScriptAddress -> hex"57",
+      Base58Type.SecretKey -> hex"d9",
+      Base58Type.ExtPublicKey -> hex"043587cf", // tprv
+      Base58Type.ExtSecretKey -> hex"04358394"  // tpub
+    )
+
+  /**
+   * Pow limit on signet
+   * [[https://github.com/bitcoin/bitcoin/blob/006d9ab9e59a00fa02b8d5f32919f9d57567fa65/src/chainparams.cpp#L316 signet pow limit]]
+   */
+  override lazy val powLimit: BigInteger = {
+    new BigInteger("00002adc28cf53b63c82faa55d83e40ac63b5f100aa5d8df62a429192f9e8ce5")
+  }
+
+  /**
+   * Minimum amount of chain work on the test network
+   */
+  override lazy val minimumChainWork: BigInteger = {
+    BigInteger.valueOf(0)
+  }
+
+  /**
+   * Regtest allows trivial difficulty blocks
+   * [[https://github.com/bitcoin/bitcoin/blob/006d9ab9e59a00fa02b8d5f32919f9d57567fa65/src/chainparams.cpp#L312 regtest min difficulty]]
+   */
+  override lazy val allowMinDifficultyBlocks: Boolean = false
+
+  /**
+   * Regtest allows pow retargetting
+   * [[https://github.com/bitcoin/bitcoin/blob/006d9ab9e59a00fa02b8d5f32919f9d57567fa65/src/chainparams.cpp#L313 signet pow retargetting]]
+   */
+  override lazy val noRetargeting: Boolean = false
```

Comment:
Interesting, i'm kind of surprised this is not `true`. So signet allows for POW retargetting? I thought signet got rid of POW all together. cc @kallewoof

Reply:
Signet in fact includes proof of work! This is, generally, kept at a very low difficulty, however (even lower than main net's lowest difficulty).